### DISCUSSION TITLE
Welcome to cfn-lint-action Discussions!

### DISCUSSION THREAD

**Discussion by ScottBrenner** [ScottBrenner/cfn-lint-action#34] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by rasovie** [comment]
Hello!  How can I configure my github action to comment when there is a error? Like the picture [link](https://raw.githubusercontent.com/ScottBrenner/cfn-lint-action/main/.github/cfn-lint-pull-request.png)

**Comment by NickLiffen** [comment]
👋 

@rasovie it should automatically do that 🤔 How are you running the action?

**Reply by rasovie** [reply]
@NickLiffen I'm using a workflow_call in a repository and this calls cfn-lint. In another repository I'm calling a action with workflow_call and in this repository the comment don't showed.

**Reply by ScottBrenner** [reply]
Someone actually emailed me about a similar issue a few weeks ago 🤔 
![image](https://user-images.githubusercontent.com/416477/138017947-86ba92d0-0032-4c8f-8aaf-2161dd7bbd2b.png)

**Comment by NickLiffen** [comment]
@ScottBrenner maybe the annotation regex isn't working, but this did use to work and I just tried and it can't get it **not** to work. Could you try and replicate this?

**Reply by ScottBrenner** [reply]
Afraid I don't have any CloudFormation to lint off-hand 😅

**Reply by ScottBrenner** [reply]
#273 is reporting this too
